documentation/asciidoc/stories/assembly_xsite_replication.adoc

```diff
@@ -1,13 +1,18 @@
 [id='configuring_backup_sites']
 :context: xsite
 = Configuring Cross-Site Replication
-{brandname} allows you to back up data across geographic locations. For
-example, you have a {brandname} cluster running on {openshift} in a data
-center in New York (SiteA). You have another {brandname} cluster running on
-{openshift} in a data center in San Francisco (SiteB).
+Set up cross-site replication to back up data between {brandname} clusters
+running in different locations.
 
-Configuring cross-site replication enables the {brandname} cluster at SiteA to
-back up data to the {brandname} cluster at SiteB.
+For example, you use the {ispn_operator} to manage a {brandname} cluster at a data center in London, **LON**, and another cluster at a data center in New York City, **NYC**. You can add **LON** and **NYC** as backup locations for one another.
```

Comment:
This sentence could lead to some confusion. Technically each site has its own operator, so there's no operator that manages multiple sites. Each sites' operator has details to connect to all locations...etc (you already mention this later on).